src/loadContract.js

```diff
@@ -3,14 +3,64 @@ const path = require('path')
 const replStarter = require('./replStarter')
 const Web3 = require('web3')
 
-module.exports = function(abiPath, address, url) {
+module.exports = function(abiPath, address, rest, url) {
   if (!url) {
     throw new Error('[loadContract] URL require')
   }
 
-  // Get abi and address from argv
+  let abisAndAddresses = [
+    {
+      abiPath: abiPath,
+      abi: parseAbi(abiPath),
+      address: address
+    }
+  ]
+
+  if (rest.length > 0) {
+    for (let i = 0; i < rest.length; i += 2) {
+      const chunk = rest.slice(i, i + 2)
+      abisAndAddresses.push({
+        abiPath: chunk[0],
+        abi: parseAbi(chunk[0]),
+        address: chunk[1]
+      })
+    }
+  }
+
+  // Connect web3
+  const web3 = new Web3(new Web3.providers.HttpProvider(url))
+
+  // Default context
+  let rplContext = {
+    web3: web3,
+    eth: web3.eth
+  }
+
+  // Add contracts into context
+  for (let contract of abisAndAddresses) {
+    const Contract = new web3.eth.Contract(contract.abi, contract.address)
+    let contractName = path.basename(contract.abiPath).split('.')[0]
+
+    if (rplContext[contractName]) {
```

Comment:
This doesn't work correctly in some veeery edge cases (for example: if you do `eth lc ERC20.abi 0x ERC201.abi 0x ERC20.abi 0x`, the suffix of the second `ERC20` will be `_2` instead of `_1`).

I don't think it's important enough to fix it: the worst case scenario is a wrong suffix in some very weird cases, but we may want to open an issue or put a comment for future reference.

Reply:
I think replacing `return key.includes(contracName)` with 
```
const [noSuffixKey] = key.split('_')
const [noSuffixContract] = contractName.split('_')
return noSuffixKey === noSuffixContract
```
would cover that edge case and others like `eth lc ERC20.abi 0x ERC20_1.abi 0x ERC20.abi 0x`, although there will be more edge cases that won't be covered